BICCN-policies/dataset-metadata.yaml

```diff
@@ -0,0 +1,85 @@
+identifier: REQUIRED ## Post upload (or during dandi organize)
+name: REQUIRED
+description: REQUIRED
+contributors: # required for author and contact
+- orcid: # REQUIRED
+  roles: # REQUIRED from https://casrai.org/credit/ + maintainer, contact
+  email: Recommended # filled from orcid
+  name: Recommended  # filled from orcid
+  affiliations: optional  # filled from orcid
+sponsors:
+- identifier: RECOMMENDED
+  name: REQUIRED
+  url: RECOMMENDED
+license:
+- url # REQUIRED
+keywords: [key1, key2,]
+consortium/project:
+- name: REQUIRED
+  identifier: REQUIRED #RRID
+associated_disease: # RECOMMENDED
+- name: REQUIRED
+  identifier: REQUIRED
+associated_anatomy: # RECOMMENDED
+- name: REQUIRED
+  identifier: REQUIRED
```

Comment:
For individual specimens we would like to enforce that anatomy comes from the sources below. HOWEVER, for description of a dataset, are there use cases where we would not want to do that? 

Mouse--http://atlas.brain-map.org/atlas?atlas=602630314
 Human (areal)--http://atlas.brain-map.org/atlas?atlas=265297126
 Human (gyral)--http://atlas.brain-map.org/atlas?atlas=138322605
 Marmoset--Use homologous Adult Human   structures, if appropriate (or until we adopt a specific marmoset ontology)